net/mwan3/files/usr/sbin/mwan3

```diff
@@ -160,7 +159,7 @@ stop()
 	done
 
 	mwan3_lock_clean
-	rm -rf "${MWAN3_STATUS_DIR}/mmx_mask"
+	rm -rf $MWAN3_STATUS_DIR
```

Comment:
We should also delete /var/run/mwan3track if mwan3 is stopped
rm -rf $MWAN3TRACK_STATUS_DIR

This variable could be defined in **/lib/mwan3/mwan3.sh** after
https://github.com/marcin1j/openwrt-packages/blob/80fb9ca76fbbc5baf0a50dd4861f0e0614f0c027/net/mwan3/files/lib/mwan3/mwan3.sh#L11